projects/plugins/jetpack/extensions/plugins/launchpad-save-modal/launchpad-save-modal.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * Launchpad Save Modal
+ *
+ * @package automattic/jetpack
+ */
+
+// Feature name.
+const FEATURE_NAME = 'launchpad-save-modal';
+
+wp_enqueue_script(
+	'launchpad-save-modal',
+	plugins_url( 'index.js', __FILE__ ),
+	array(),
+	JETPACK__VERSION,
+	true
+);
+
+wp_localize_script(
+	'launchpad-save-modal',
+	'currentSite',
```

Comment:
If we stick with `wp_localize_script`, I would want to change the `currentSite` string here to something else. This is creating `window.currentSite` in the global JS namespace and could easily collide with another generically named global. So we usually pick something much less generic, prefix it with an underscore, or both.

So here I'd choose `LaunchPadModalOptions` or `_launchpadModalOptions` or something along those lines.